lib/kdump_utils.pm

```diff
@@ -92,15 +106,17 @@ sub prepare_for_kdump {
     $args{test_type} //= '';
 
     # disable packagekitd
-    quit_packagekit;
-    my @pkgs = qw(yast2-kdump kdump);
-    push @pkgs, qw(crash);
+    if (!is_transactional) {
```

Comment:
Maybe it deserves separate function `get_repo_url_for_kdump_transactiona`  as well.